libminifi/src/capi/Plan.cpp

```diff
@@ -23,19 +23,8 @@
 #include <set>
 #include <string>
 
-bool intToFailureStragey(int in, FailureStrategy *out) {
```

Comment:
This was just forgotten in the previously merged PR, the declaration was already removed from the header.